samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/evaluation/ExportResponsesBean.java

```diff
@@ -204,25 +209,31 @@ public void exportExcel(ActionEvent event){
         ArrayList<Object> headerList = new ArrayList<Object>();
         headerList.add(HEADER_MARKER);
         if (anonymous) {
-  		  headerList.add(ContextUtil.getLocalizedString("org.sakaiproject.tool.assessment.bundle.EvaluationMessages","sub_id"));
+  		  headerList.add(ContextUtil.getLocalizedString(MSG_BUNDLE,"sub_id"));
   	  	}
   	  	else {
-  		  headerList.add(ContextUtil.getLocalizedString("org.sakaiproject.tool.assessment.bundle.EvaluationMessages","last_name"));
-  		  headerList.add(ContextUtil.getLocalizedString("org.sakaiproject.tool.assessment.bundle.EvaluationMessages","first_name"));
-  		  headerList.add(ContextUtil.getLocalizedString("org.sakaiproject.tool.assessment.bundle.EvaluationMessages","user_name"));
-  		  headerList.add(ContextUtil.getLocalizedString("org.sakaiproject.tool.assessment.bundle.EvaluationMessages","num_submission"));
+  		  headerList.add(ContextUtil.getLocalizedString(MSG_BUNDLE,"last_name"));
+  		  headerList.add(ContextUtil.getLocalizedString(MSG_BUNDLE,"first_name"));
+  		  headerList.add(ContextUtil.getLocalizedString(MSG_BUNDLE,"user_name"));
+  		  headerList.add(ContextUtil.getLocalizedString(MSG_BUNDLE,"num_submission"));
   	  	}
 
+        if (ServerConfigurationService.getBoolean(SamigoConstants.SAK_PROP_EXPORT_INCLUDE_DATES,
+                SamigoConstants.SAK_PROP_EXPORT_INCLUDE_DATES_DEFAULT)) {
```

Comment:
Why would someone want this feature disabled? What's the use case for an institution wanting to disable this?

Reply:
I don't think there is a specific use case we had in mind, just in general we try to add new features behind properties to offer flexibility. Potentially there could be workflows that make use of the exported files and changing the format of these files could cause problems with that.

Reply:
If we can't think of a single use case for not exporting it, then preference will be for smaller change. Smaller changes mean less chance of a regression, less new documentation needed, etc

If it's a good change, let's just make the good change.

Reply:
Well, we're changing the format of the export file so that will have to be documented anyway. We don't know all the external workflows that might make use of this file, so we can't really have a good idea of the problems that people may have with the change. I've had to modify plenty of code introduced in new Sakai versions that we didn't want, because it wasn't behind a property.

In this case, reading a single property value with a default doesn't introduce a lot of possibility for regressions.

Reply:
I'm not a fan of Western patches that introduce new interfaces hidden behind properties. Introducing more code and more interfaces (e.g., Evaluations -> "New style report") is just the wrong direction IMO. I am always in favor of making positive changes that have community consensus.

If you feel that too many changes that are not positive are making it into Sakai, the only thing I can recommend is to review more Github PRs before they get committed.

Reply:
Not everyone has the same idea of what is a positive change. Properties at least give you the option to easily opt out if you don't agree.

Anyway, I'll remove the property if it is going to be a problem.